src/web/app/lib/auth.ts

```diff
@@ -1,12 +1,22 @@
 import NextAuth, { Profile, User as NextAuthUser } from "next-auth";
 import { OAuthConfig } from "next-auth/providers";
+import { DefaultAzureCredential } from "@azure/identity";
+import { SecretClient } from "@azure/keyvault-secrets";
 
 // Function to convert PEM to CryptoKey
 async function pemToPrivateKey(): Promise<CryptoKey> {
-  const pem = process.env.AUTH_NHSLOGIN_PRIVATE_KEY;
+  var pem;
```

Comment:
I think you can get rid of this and just have 

`const pem = secret.value;` on line 16

Also we use let & const for variables in ES6

> While let and const are block-scoped, var declarations are either globally scoped or function-scoped. Let variables can be updated but not re-declared, const variables cannot be updated or re-declared, and var variables may both be updated and re-declared inside their scope.